src/GraphQL/Execution/IExecutionContext.cs

```diff
@@ -7,6 +7,9 @@
 
 namespace GraphQL.Execution
 {
+    /// <summary>
+    /// Provides information regarding the currently executing document.
```

Comment:
executed?

Reply:
the context is (typically) only used while the document is being executed, not afterwards.

Reply:
right